03_Data_Management/General.Rmd

```diff
@@ -8,15 +8,47 @@ knitr::include_graphics("images/network/Data Management_General.png")
 
 ### Sort
 
+Sorting out features in ascending or descending order seems a quite primitive operation in any programming language or software package. Even though it is indeed primitive, it is also quite important for filtering and cleaning our datasets. In ArcGIS pro this operation is performed using the tool `Sort`, which is part of the General toolset of the Data Managenent Toolbox. Below we present how we can perform the aforementioned operation using R. For our example we use the Simple Feature object _bezirke_, which depicts the districts within the country of Switzerland Furthermore for performing the sorting operation, we use the column that represents the area (in square km) of every of the districts. 
+
+```{r sort_BaseR}
+library(sf)
+library(arc2r)
+# Read the dataset depicting the districts (Bezirke) in the country of Switzerland
+bezirke <- arc2r::bezirke
```

Comment:
This is a minor thing, but I was thinking about how we should "get" the data from the arc2r package. Basically, you could just call `bezirk` without "getting" it first, but this might be a bit confusing for the user.

I woud suggest using `data("bezirke")`, what do you think Nikos?

Reply:
@ratnanil that's true. Maybe we can specify a proper/specific way to retrieve the data within the readme file -- so we can be consistent and also the user can easily grasp the idea of how to get the data

Reply:
I suggest using `data("bezirke")` and keep this consistent